lib/Semantics/tools.cpp

```diff
@@ -17,6 +17,7 @@
 #include "flang/Semantics/symbol.h"
 #include "flang/Semantics/tools.h"
 #include "flang/Semantics/type.h"
+#include "llvm/Support/raw_ostream.h"
 #include <algorithm>
 #include <set>
 #include <sstream>
```

Comment:
Can we remove this header?

Reply:
According to our talks it will need, because of this: 
 llvm::raw_string_ostream ss {buf};
ss << "node has not been analyzed:\n";

Reply:
I meant the sstream header here not the llvm one